tests/test_country_config.py

```diff
@@ -94,7 +94,63 @@ def test_validate_codab_layer_base_name(mock_parse_yaml):
         create_country_config(iso3="abc")
 
 
-def test_validate_fewsnet_region_name(mock_parse_yaml):
+def test_codab_validate_admin_level_max(mock_parse_yaml):
+    """Test that layer basename requires correct placeholder."""
```

Comment:
It looks like below you're only testing for the `admin_level` not actually about the placeholder. I think it's accidentally the same base code as the next test.

Reply:
Oops, some code copy-pasting gone terribly awry, fixed